wled00/button.cpp

```diff
@@ -29,7 +29,7 @@ void shortPressAction(uint8_t b)
 #ifndef WLED_DISABLE_MQTT
   // publish MQTT message
   if (buttonPublishMqtt && WLED_MQTT_CONNECTED) {
-    char subuf[64];
+    char subuf[MQTT_MAX_TOPIC_LEN + 32];
```

Comment:
maybe this should be `MQTT_MAX_TOPIC_LEN * 2`  ? because we have two "topic" inputs: _mqtt_topic_button, mqttDeviceTopic